pcomfortcloud/__main__.py

```diff
@@ -46,13 +50,7 @@ def main():
     parser.add_argument(
         '-t', '--token',
         help='File to store token in',
-        default='~/.pcomfortcloud-token')
-
-    parser.add_argument(
-        '-s', '--skipVerify',
-        help='Skip Ssl verification if set as True',
-        type=str2bool, nargs='?', const=True,
-        default=False)
+        default='token.json')
```

Comment:
Is this a consern? Is it better to store the file locally or in the home directory?

Reply:
I've also noticed we have gone from file location to file content when passing data to `Session`. Maybe I'll turn this around again for `Session` (backward compatible), but keeping it to file content in `Authentication`. 

Any thoughts?